addons/sourcemod/scripting/fuckZones.sp

```diff
@@ -146,6 +147,10 @@ public Plugin myinfo =
 
 public APLRes AskPluginLoad2(Handle myself, bool late, char[] error, int err_max)
 {
+	EngineVersion engine = GetEngineVersion();
+	if (engine == Engine_Left4Dead || engine == Engine_Left4Dead2)
+		g_bIsL4D = true;

```

Comment:
I think caching return of `GetEngineVersion` in `g_bIsL4D` is not really needed for one usage, because `OnPluginStart` will just triggered one time on start up or reload.